### ISSUE TITLE
Can't get it working on iOS 14

### ISSUE THREAD

**Issue by souriscloud** [triniwiz/nativescript-image-cache-it#83]
Is this plugin ready for NS7? Since I've read some text about that this plugin has to work with NS7, so I decided to implement it in our project. But I can't get it working on iOS, it immediately crashes the app with error about Vue integration for ios, some symbol is there undefined.

**Comment by souriscloud** [comment]
```
(CoreFoundation) *** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'prototype' of undefined
at
register(file: src/packages/core/ui/core/properties/index.ts:786:28)
at ../node_modules/nativescript-image-cache-it/image-cache-it.common.js(file: node_modules/nativescript-image-cache-it/image-cache-it.common.js:239:23)
at __webpack_require__(file: src/webpack/bootstrap:816:0)
at fn(file: src/webpack/bootstrap:120:0)
at ../node_modules/nativescript-image-cache-it/image-cache-it.js(file: node_modules/nativescript-image-cache-it/image-cache-it.ios.js:6:13)
at __webpack_require__(file: src/webpack/bootstrap:816:0)
at fn(file: src/webpack/bootstrap:120:0)
at (file: src/main.js:1:0)
at ./main.js(file:///app/bundle.js:17419:30)
at __webpack_require__(file: src/webpack/bootstrap:816:0)
at checkDeferredModules(file: src/webpack/bootstrap:43:0)
at webpackJsonpCallback(file: src/webpack/bootstrap:30:0)
at (file:///app/bundle.js:2:57)
at require(:1:137)
', reason: '(null)'
*** First throw call stack:
(
0   CoreFoundation                      0x00007fff2041daf2 __exceptionPrep<…>
Successfully synced application rs.kupoman.app on device 729D8FB7-4367-4DEA-AFC7-73149065AC7B.

```

**Comment by triniwiz** [comment]
Use `@triniwiz/nativescript-image-cache-it`


Sent from Yahoo Mail for iPhone


On Tuesday, December 22, 2020, 10:28 AM, Lukas Novotny <notifications@github.com> wrote:



(CoreFoundation) *** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'prototype' of undefined
at
register(file: src/packages/core/ui/core/properties/index.ts:786:28)
at ../node_modules/nativescript-image-cache-it/image-cache-it.common.js(file: node_modules/nativescript-image-cache-it/image-cache-it.common.js:239:23)
at __webpack_require__(file: src/webpack/bootstrap:816:0)
at fn(file: src/webpack/bootstrap:120:0)
at ../node_modules/nativescript-image-cache-it/image-cache-it.js(file: node_modules/nativescript-image-cache-it/image-cache-it.ios.js:6:13)
at __webpack_require__(file: src/webpack/bootstrap:816:0)
at fn(file: src/webpack/bootstrap:120:0)
at (file: src/main.js:1:0)
at ./main.js(file:///app/bundle.js:17419:30)
at __webpack_require__(file: src/webpack/bootstrap:816:0)
at checkDeferredModules(file: src/webpack/bootstrap:43:0)
at webpackJsonpCallback(file: src/webpack/bootstrap:30:0)
at (file:///app/bundle.js:2:57)
at require(:1:137)
', reason: '(null)'
*** First throw call stack:
(
0   CoreFoundation                      0x00007fff2041daf2 __exceptionPrep<…>
Successfully synced application rs.kupoman.app on device 729D8FB7-4367-4DEA-AFC7-73149065AC7B.

**Comment by RayburnTrading** [comment]
Do you have a working NS8/Vue version?

If I install @triniwiz/nativescript-image-cache-it I don't have a /vue import and if I install nativescript-image-cache-it I get the prototype error above?

I'm using an Android device.

**Comment by souriscloud** [comment]
@RayburnTrading I actually have it working on NS8-Vue.

import in main.js as
```javascript
import { ImageCacheIt } from '@triniwiz/nativescript-image-cache-it'
````

then register element like 
```javascript
// ImageCacheIt
Vue.registerElement('ImageCacheIt', () => require('@triniwiz/nativescript-image-cache-it').ImageCacheIt)
```

any other things follow ur specific usage and docs supplied :) for example i need access to cache object so I create one and then make accessor function as built-in prototype of Vue instance, to reach cache object instance in vue component by `this.$getImageCacheIt()` which gives me instance object of that cache. But you would not need it? IDK right now, i have implemented this few months ago

**Comment by triniwiz** [comment]
I’ve fixed the missing /vue directory in 7.0.7


Sent from Yahoo Mail for iPhone


On Thursday, May 20, 2021, 12:05 PM, Lukas Novotny ***@***.***> wrote:




@RayburnTrading I actually have it working on NS8-Vue.

import in main.js as
import { ImageCacheIt } from ***@***.***/nativescript-image-cache-it'
then register element like
// ImageCacheIt
Vue.registerElement('ImageCacheIt', () => ***@***.***/nativescript-image-cache-it').ImageCacheIt)
any other things follow ur specific usage and docs supplied :) for example i need access to cache object so I create one and then make accessor function as built-in prototype of Vue instance, to reach cache object instance in vue component by this.$getImageCacheIt() which gives me instance object of that cache. But you would not need it? IDK right now, i have implemented this few months ago
